Update to readme with more configuration, update and delete instructions (#146)
shibayan/keyvault-acmebot: README.md

- Acmebot:Webhook
  - Webhook destination URL (optional, Slack and Microsoft Teams are recommended)

There are also additional settings that will be automatically created by Key Vault Acmebot
- Acmebot:Endpoint

>   - The letsencrypt endpoint used to renew certificates

Reviewer: ```suggestion
  - The ACME endpoint used to issue certificates
```

- Acmebot:Contacts

>   - The email address (required) used in letsencrypt requests

Reviewer: ```suggestion
  - The email address (required) used in ACME account registration
```

- Acmebot:AzureDns:SubscriptionId

>   - Your Azure subscription ID in order to identify which DNS resources can be used for let's encrypt

Reviewer: ```suggestion
  - Your Azure subscription ID in order to identify which DNS resources can be used for ACME
```

The issued certificate can be downloaded from Key Vault and used elsewhere, either in Azure or outside Azure.

#### Remove a Certificate

> To Remove a certificate from the system delete it from the key vault. Key Vault Acmebot
>  will no longer renew the certificate.

Reviewer: ```suggestion
To Remove a certificate from the system delete it from the Key Vault. Key Vault Acmebot will no longer renew the certificate.
```

#### Reinstalling Or Updating Key Vault Acmebot

> To Reinstall or Upgrade Key Vault Acmebot without removing your certificates, ensure that the keyvault is not removed. Key Vault Acmebot will use the exisiting certificates and vault after upgrade or reinstall

Reviewer: ```suggestion
To Reinstall or Upgrade Key Vault Acmebot without removing your certificates, ensure that the Key Vault is not removed. Key Vault Acmebot will use the exisiting certificates and vault after upgrade or reinstall
```
